src/open_inwoner/configurations/bootstrap/kic.py

```diff
@@ -0,0 +1,176 @@
+                "auth_type": AuthTypes.zgw,
+                "client_id": settings.CONTACTMOMENTEN_API_CLIENT_ID,
+                "secret": settings.CONTACTMOMENTEN_API_SECRET,
+                "user_id": settings.CONTACTMOMENTEN_API_CLIENT_ID,
+                "user_representation": org_label,
+            },
+        )
+        if not created:
+            contactmomenten_service.oas = settings.CONTACTMOMENTEN_API_ROOT
+            contactmomenten_service.client_id = settings.CONTACTMOMENTEN_API_CLIENT_ID
+            contactmomenten_service.secret = settings.CONTACTMOMENTEN_API_SECRET
+            contactmomenten_service.user_id = settings.CONTACTMOMENTEN_API_CLIENT_ID
+            contactmomenten_service.user_representation = org_label
+            contactmomenten_service.save()
+
+    def test_configuration(self):
+        """
+        actual testing is done in final step
+        """
+
+
+class KICAPIsConfigurationStep(BaseConfigurationStep):
+    """
+    Configure the KIC settings and set any feature flags or other options if specified
+    """
+
+    verbose_name = "Klantinteractie APIs configuration"
+    enable_setting = "KIC_API_CONFIG_ENABLE"
+
+    def is_configured(self) -> bool:
+        kic_config = OpenKlantConfig.get_solo()
+        return bool(kic_config.klanten_service) and bool(
+            kic_config.contactmomenten_service
+        )
+
+    def configure(self):
+        config = OpenKlantConfig.get_solo()
+        config.klanten_service = Service.objects.get(api_root=settings.KLANTEN_API_ROOT)
+        config.contactmomenten_service = Service.objects.get(
+            api_root=settings.CONTACTMOMENTEN_API_ROOT
+        )
+
+        if settings.KIC_REGISTER_EMAIL:
+            config.register_email = settings.KIC_REGISTER_EMAIL
+        if settings.KIC_REGISTER_CONTACT_MOMENT is not None:
+            config.register_contact_moment = settings.KIC_REGISTER_CONTACT_MOMENT
+        if settings.KIC_REGISTER_BRONORGANISATIE_RSIN:
+            config.register_bronorganisatie_rsin = (
+                settings.KIC_REGISTER_BRONORGANISATIE_RSIN
+            )
+        if settings.KIC_REGISTER_CHANNEL:
+            config.register_channel = settings.KIC_REGISTER_CHANNEL
+        if settings.KIC_REGISTER_TYPE:
+            config.register_type = settings.KIC_REGISTER_TYPE
+        if settings.KIC_REGISTER_EMPLOYEE_ID:
+            config.register_employee_id = settings.KIC_REGISTER_EMPLOYEE_ID
+        if settings.KIC_USE_RSIN_FOR_INNNNPID_QUERY_PARAMETER is not None:
+            config.use_rsin_for_innNnpId_query_parameter = (
+                settings.KIC_USE_RSIN_FOR_INNNNPID_QUERY_PARAMETER
+            )
+
+        config.save()
+
+    def test_configuration(self):
+        """
+        make requests to the APIs and verify that a connection can be made
+        """
+        klanten_client = build_client("klanten")
+        contactmomenten_client = build_client("contactmomenten")
```

Comment:
I've noticed that we somtimes use `contactmomenten_client` (plural) and sometimes `contactmoment_client` (singular). It would be good to make this consistent so that it is easier to grep on the variable.